bigbluebutton-html5/imports/api/users/server/modifiers/setChangedSettings.js

```diff
@@ -0,0 +1,30 @@
+import { check } from 'meteor/check';
+import Users from '/imports/api/users';
+import Logger from '/imports/startup/server/logger';
+
+export default function setChangedSettings(userId, setting, value) {
+  check(userId, String);
+  check(setting, Match.Maybe(String));
```

Comment:
`setting` was checked as a String in  bigbluebutton-html5/imports/api/users/server/methods/userChangedSettings.js 

Why would it be a Maybe (String) here? We likely don't need to check it again either way

Reply:
You are right, updated.